modules/insights-operator-what-information-is-collected.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/about-insights-operator.adoc
+
+[id="insights-operator-what-information-is-collected_{context}"]
+= Information collected by the Insights Operator
+
+The Insights Operator collects the following information:
+
+* The version of the cluster and its components
+* Channel and image repository used for an update
+* Details about errors that occurred in the cluster components
+* Progress information of an update that is running
+* Details about the {product-title} cluster components configuration
+* Details about disabled {product-title} cluster features
+* Information about enabled authentication mechanisms
+* A unique random identifier that is generated during installation
+* The name of the platform that {product-title} is deployed on, such as Amazon Web Services
```

Comment:
Change this to "Information about the platform" rather than "the name" (since we gather more)